### DISCUSSION TITLE
What to type in for git CLI stdin wrapper after pressing PUSH !? :(

### DISCUSSION THREAD

**Discussion by DeMichaelis** [gitextensions/gitextensions#9889] [Q&A]
Hi @ all
Short Question: What do i have to type here ? It from git CLI stdin wrapper.
I clicked PUSH in the top Toolbar and then the PUSH Button in the new Window.
But i dont know what the program is asking for !? Doesnt matter what i type in there. I can not push!
![Ashampoo_Snap_Montag, 7  März 2022_10h42m32s_024_](https://user-images.githubusercontent.com/61233948/157007691-92e4e698-21ff-4e64-a611-6dba2c6a7a83.png)

**Comment by gerhardol** [comment]
The popup is from the Git Credential helper *git CLI stdin wrapper* you have installed, not from GE.

**Reply by DeMichaelis** [reply]
Ok, yes. Sorry. i forgot to mention. i edited it. thx.
But it doesnt matter what i type in there. i cant push.

**Reply by DeMichaelis** [reply]
noone here with an idea ? :(

**Reply by pmiossec** [reply]
Your ssh passphrase?

If whatever you try is not working, use the command line to push and see if it works (to diagnose if it is GitExtensions which is not well configured or has a problem). Or if it is git which is not well configured.

Authentication handling is always a little tricky and we don't configure it often so it is difficult to help 😕

**Reply by DeMichaelis** [reply]
i tried my Git SSH Key.... but it wasnt successful :(

![image](https://user-images.githubusercontent.com/61233948/161218721-6e5b408e-2679-4fd0-b5d4-3a52043b772f.png)

**Reply by pmiossec** [reply]
And the result of the second (and more important) test? I.e testing from the command line?

**Reply by DeMichaelis** [reply]
i was successful !
when i changed the settings -> SSH to "OpenSSH" it worked.
thanks to all. :)
